htsget_ingest.py

```diff
@@ -258,6 +260,36 @@ def htsget_ingest(ingest_json, do_not_index=False):
         for url in to_index:
             response = requests.get(url, headers=headers, params={"do_not_index": do_not_index})
 
+    # update completeness stats for program_ids with created samples
+    statistics = {}
+    for program_id in program_ids:
+        url = f"{HTSGET_URL}/htsget/v1/samples"
```

Comment:
This does work on my machine, but for curiosity's sake -- how are we certain that HTSGet has finished ingesting at this point? Or do the above calls not return until the samples' files have finished being uploaded?

Reply:
The actual ingest is very fast and synchronous; there isn't anything to upload (it's just links) and we check the status codes as they come through. The thing that is async is indexing.